efficientdet/keras/utils_keras.py

```diff
@@ -52,11 +52,11 @@ def build_batch_norm(is_training_bn: bool,
   axis = 1 if data_format == 'channels_first' else -1
   batch_norm_class = utils.batch_norm_class(is_training_bn, strategy)
   bn_layer = batch_norm_class(axis=axis,
-                              momentum=momentum,
-                              epsilon=epsilon,
-                              center=True,
-                              scale=True,
-                              gamma_initializer=gamma_initializer,
-                              name=name)
+                            momentum=momentum,
```

Comment:
Revert these changes.

You may want to use the linter to check format: https://github.com/google/automl/blob/master/efficientdet/g3doc/faq.md#21-how-can-i-format-my-code-for-prs

Reply:
Thanks, I will apply the linter to check format.